helpscout/hsds-react
# PROD-390 Add scheme to urls if missing

Jira: https://helpscout.atlassian.net/browse/PROD-390

Problem: 
Websites that were missing `http://` or `https://` wouldn't open if you clicked the "new window" button. 

Solution:
Check for the presence of the scheme and add if missing.

To test:
In Storybook, check that the "new window" button in the EditableField URL story will open for websites with and without the `http://` scheme.

Note:
We had similar code inside the `convertLinksToHTML` util function, so I abstracted it into one new util and updated both `convertLinksToHTML` and `EditableField` to make use of it. 

Using a beta release, I checked that the following links opened as expected:
```
google.com
www.google.com
http://www.google.com
https://www.google.com
```
